neqo-transport/src/cc.rs

```diff
@@ -87,22 +105,44 @@ impl CongestionControl {
             assert!(self.bytes_in_flight >= pkt.size);
             self.bytes_in_flight -= pkt.size;
 
-            if self.in_congestion_recovery(pkt.time_sent) {
+            if self.in_congestion_recovery(pkt.time_sent, PacketState::Acked) {
```

Comment:
OK, this is a functionality change.  This clears the recovery timer.  I think that's right, but I'm concerned that I can't find a test that has updated as a result of the change.  That means that we weren't testing the exit from congestion recovery properly.